src/main/java/baritone/behavior/InventoryBehavior.java

```diff
@@ -97,7 +97,8 @@ private void swapWithHotBar(int inInventory, int inHotbar) {
     private int firstValidThrowaway() { // TODO offhand idk
         NonNullList<ItemStack> invy = ctx.player().inventory.mainInventory;
         for (int i = 0; i < invy.size(); i++) {
-            if (Baritone.settings().acceptableThrowawayItems.value.contains(invy.get(i).getItem())) {
+            Item item = invy.get(i).getItem();
+            if (MovementHelper.filterThrowawayForPlacing(Baritone.settings().acceptableThrowawayItems.value).contains(item)) {
```

Comment:
Probably better for this to be run once at the beginning of the function instead of for every item in the inventory.